README.rst

```diff
@@ -71,13 +71,59 @@ Allows changing configuration details of a given repository
 Installation
 ============
 
-From System Package Manager
----------------------------
+From System Repository 
+----------------------
 
 If your operating system packages repolib, you can install it by running::
-    
+
     sudo apt install python3-repolib
 
+From Pre-Compiled .deb Package
+------------------------------
+
+If your distro does not yet supply the package, you can install the
+`latest release`__ using the script below (At the time of writing this,
```

Comment:
The parenthetical not at the beginning of a sentence needs to have its first word un-capitalized. The current wording is a little overpersonal anyway, so please replace it with this:

> (replace `2.0.0` with the version you wish to install):

Reply:
Shouold be better now.